### ISSUE TITLE
ItemLine doesn't appear on paper 1.8.8

### ISSUE THREAD

**Issue by Alejandro12120** [unldenis/Hologram-Lib#8]
Hello, I am using the latest version of Paper 1.8.8 and the latest version of Hologram-Lib (`1.2.6`).

And I do not why but my item line doesn't appear.
This is my code:
```java
Hologram.Builder builder = Hologram.builder()
                        .location(location.clone().add(0.5, -1.0, 0.5))
                        .addPlaceholder("<playerName>", Player::getName);

builder.addLine(new ItemStack(Material.FISHING_ROD));
builder.addLine(Common.colorize("&7A item should appear above"));
this.holograms.add(builder.build(pool));
```
Image:
![image](https://user-images.githubusercontent.com/45500329/162044294-0e8c3a72-e7de-4a92-aeee-f62cea1abc93.png)

**Comment by unldenis** [comment]
Have you tried to put like a golden block?

**Comment by Alejandro12120** [comment]
Looks like it only works with blocks. Although the block doesn't spin.

![image](https://user-images.githubusercontent.com/45500329/163735014-92f86bc8-eddc-46c3-b284-c09b220b7f40.png)

**Comment by Alejandro12120** [comment]
And this code doesn't make the block spin

```java
Hologram.Builder builder = Hologram.builder()
                        .location(location.clone().add(0.5, -1.0, 0.5))
                        .addPlaceholder("<playerName>", Player::getName);

builder.addLine(new ItemStack(Material.GOLD_BLOCK));
builder.addLine(Common.colorize("&7A item should appear above"));

Hologram hologram = builder.build(pool);
hologram.getLines().stream().filter(line -> line instanceof ItemLine).forEach(line -> line.setAnimation(Animation.CIRCLE));

this.holograms.add(hologram);
```

**Comment by unldenis** [comment]
> And this code doesn't make the block spin

Fixed since 1.2.8.
However, the items are temporarily not possible, as they cannot be set as a helmet

**Comment by unldenis** [comment]
Since 2.6.0 items are possible.
